src/main/java/org/spongepowered/common/mixin/core/entity/passive/MixinEntityOcelot.java

```diff
@@ -80,6 +83,13 @@ public int onTame(Random rand, int bound, EntityPlayer player, EnumHand hand) {
         return 1;
     }
 
+    @Inject(method = "processInteract", locals = LocalCapture.CAPTURE_FAILHARD,
+            at = @At(value = "FIELD", target = "Lnet/minecraft/entity/player/PlayerCapabilities;isCreativeMode:Z"))
+    public void onHealFeed(EntityPlayer player, EnumHand hand, CallbackInfoReturnable<Boolean> cir, ItemStack itemStack) {
```

Comment:
Yeah, but since you don't want to override or access it anywhere except in this class there is no reason to keep it public. I'm not entirely sure if Mixin renames injection methods now but a while back this could actually cause really weird bugs if you had something like `onInit` in multiple mixins and it was actually overriding the injection method in super classes.